web/src/components/HeaderLogo.tsx

```diff
@@ -18,7 +18,7 @@ const LogoContainer = styled.div`
   display: flex;
   justify-content: start;
   align-items: center;
-  flex: 0 1 220px;
+  flex: initial;
```

Comment:
tbh couldn't see that this is needed! I think you can remove it

Reply:
Without this it looks like this for me on firefox: 
<img width="636" alt="Bildschirmfoto 2023-07-27 um 10 35 52" src="https://github.com/digitalfabrik/integreat-app/assets/28013368/84c86507-efc8-4cc1-8faa-29a903478450">
And with `flex: initial` it looks like this:
<img width="636" alt="Bildschirmfoto 2023-07-27 um 10 35 52" src="https://github.com/digitalfabrik/integreat-app/assets/28013368/7b325438-b38c-4d55-a175-8844aa3c3a5c">